nondex-maven-plugin/src/it/clean-it with-whitespace-in-path/pom.xml

```diff
@@ -23,7 +32,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
-        <version>2.20</version>
+        <version>2.22.2</version>
```

Comment:
This is optional: you may want to use something like `<surefire.version>2.22.2</surefire.version>` in `properties`, and then use `<version>${surefire.version}</version>` for all occurrences.

And I am not sure whether using a more recent version would help, as discussed in https://github.com/TestingResearchIllinois/NonDex/pull/154#discussion_r536483864

Reply:
You're right! We don't necessarily need to do this upgrade for `Surefire` here. I'll revert the changes here in the break-down PR.